apps/examples/testcase/ta_tc/systemio/itc/itc_gpio.c

```diff
@@ -20,295 +20,325 @@
 
 /// @brief Test Case Example for gpio API
 
-#include <tinyara/config.h>
-#include <stdio.h>
-#include <stdlib.h>
-#include <errno.h>
-#include <tinyara/time.h>
-#include <sys/time.h>
 #include "itc_internal.h"
 #include <iotbus_gpio.h>
-#include <iotbus_error.h>
 #ifdef CONFIG_ARCH_CHIP_STM32
```

Comment:
To use CONFIG_XX, you should keep including <tinyara/config.h>

Reply:
Okay, I included <tinyara/config.h>